app/lib/upvs_submissions/forms/fuzs_data.rb

```diff
@@ -248,6 +249,16 @@ def with_missing_municipality_identifier?
 
         private
 
+        def parse_person_formatted_name(formatted_name)
+          return if @person_prefixes.present? || @person_postfixes.present?
+
+          formatted_name_pattern = /(.*)\b#{given_name}\b\s\b#{family_name}\b(.*)/
+          prefixes, postfixes = formatted_name.match(formatted_name_pattern).captures
```

Comment:
Tu nestaci `split`?

Reply:
Split ako, podla coho @jsuchal?

Reply:
Aha, ono to asi funguje rovnako. Ja som myslel nieco taketo `prefix, suffix = s.split(/Jan Suchal/, 2)`